### ISSUE TITLE
Future timeout exception when db is down

### ISSUE THREAD

**Issue by semberal** [semberal/dbstress#25]
When a database is down, a future timeout exception is exported instead of the raw SQL exception the user is probably interested in.

**Comment by semberal** [comment]
It only happens when connection init timeout is configured, which is ok. We should just show more user friendly message.

**Comment by semberal** [comment]
Fixed by #31. Currently, the output looks like this:

```
13:55:43.237 [ERROR] Cannot initialize database connection: Database connection initialization timeout (1 ms)
13:55:43.246 [ERROR] Some database connection could not be initialized, terminating the scenario
```
